src/components/BatchClassComponents/DoubtClassExerciseComponent.js

```diff
@@ -0,0 +1,349 @@
+                variant="outlined"
+                color="secondary"
+                style={{
+                  borderRadius: 90,
+                  height: 30,
+                  backgroundColor: "#DADAEC",
+                }}
+              />
+
+              <Chip
+                label={lang[actions?.lang]}
+                variant="outlined"
+                color="secondary"
+                style={{ marginLeft: 10, borderRadius: 90, height: 30 }}
+              />
+            </Box>
+            <Typography variant="body1">
+              Clear your doubts related to the first class of Python and other
+              queries during your studies
+            </Typography>
+            <Typography
+              variant="body"
+              mt={2}
+              style={{
+                display: "flex",
+                padding: "10px 0",
+              }}
+            >
+              <img
+                className={classes.icons}
+                src={require("./assets/calender.svg")}
+                alt="Students Img"
+              />
+              {actions?.start_time
+                ? format(actions?.start_time, "dd MMM yy")
+                : ""}
+              ,
+              {actions?.start_time
+                ? format(actions?.start_time, "hh:mm aaa")
+                : ""}{" "}
+              -{actions?.end_time ? format(actions?.end_time, "hh:mm aaa") : ""}
+            </Typography>
+            <Typography
+              variant="body1"
+              mb={2}
+              style={{
+                display: "flex",
+              }}
+            >
+              {" "}
+              <img
+                className={classes.icons}
+                src={require("./Revision/assets/Group.svg")}
+                alt="Students Img"
+              />
+              {actions?.facilitator_name}
+            </Typography>
+            <Typography variant="body1" color="text.secondary" mb={2}>
+              Please join at least 10 mintues before the scheduled time
+            </Typography>
+            {isEnrolled ? (
+              <>
+                {TimeLefts == "joinNow" ? (
+                  <a
+                    style={{
+                      textDecoration: "none",
+                    }}
+                    href={actions?.meet_link}
+                    target="_blank"
+                  >
+                    <Button
+                      variant="contained"
+                      style={{ marginTop: 20 }}
+                      fullWidth
+                    >
+                      Join Now
+                    </Button>
+                  </a>
+                ) : (
+                  <Button
+                    disabled={true}
+                    variant="contained"
+                    sx={{ fontSize: "1rem" }}
+                    fullWidth
+                    style={{ marginTop: 20 }}
+                  >
+                    Starts in {TimeLefts}
+                  </Button>
+                )}
```

Comment:
Another place we could use the `ClassJoinTimerButton` component

Reply:
@jschanker will update that by today